src/main.ts

```diff
@@ -41,16 +42,13 @@ app.get('/.well-known/server-health', (req, res) => {
   res.status(200).send('ok');
 });
 
-// TODO: return appropriate cache headers here
-// let's start with 30 minutes
-// need to research exact header names
-app.get('/', async (req, res) => {
+app.get('/:scheduledSurfaceID/:date', async (req, res) => {
   // enable 30 minute cache when in AWS
   if (config.app.environment !== 'development') {
     res.set('Cache-control', 'public, max-age=1800');
   }
 
-  res.json({ Hello: 'World' });
+  res.json(await getStories(req.params.date, req.params.scheduledSurfaceID));
```

Comment:
not a blocker for this PR, but should we do any validation on the `date` parameter? could do a regex right in the url definition...

Reply:
I did ask for feedback on that in the PR! :) I've added date validation to the ticket that should limit the data to Pocket Hits surfaces only: https://getpocket.atlassian.net/browse/BACK-1393